client/my-sites/upgrades/index.js

```diff
@@ -284,10 +285,19 @@ module.exports = function() {
 		);
 
 		page(
-			'/checkout/:domain/:plan_name?',
+			'/checkout/:domain/:planName?',
 			adTracking.retarget,
 			controller.siteSelection,
 			upgradesController.checkout
 		);
+
+		page(
+			'/start-trial/:planName/:domain',
```

Comment:
Have another look to the file you just linked and you'll see why I proposed this change. It contains routes for the domain management section and most of them use `:site` **and** `:domain` because we have to make the distinction between the site selected by the user and the domain being managed. Further down this file we have routes for the domain search section and this is where we got it wrong (it was one of the first section of the store migrated to Calypso). We should fix these but in the meantime let's try to use the right parameter for new routes :).